x-pack/plugins/security_solution_serverless/server/common/services/usage_reporting_service.test.ts

```diff
@@ -24,6 +24,8 @@ describe('UsageReportingService', () => {
   let usageApiConfig: UsageApiConfigSchema;
   let service: UsageReportingService;
 
+  const kibanaVersion = '8.16.0';
```

Comment:
Will this always be static?
if wanting to use the "current stack version", then you could get that from `import { kibanaPackageJson } from '@kbn/repo-info';`